suites/pacific/rbd/tier-2_rbd_regression.yaml

```diff
@@ -168,12 +168,6 @@ tests:
       name: Test image migration on ecpool with different K_m values
       polarion-id: CEPH-83573322
 
-  - test:
```

Comment:
why this test got deleted?

Reply:
This been deleted and added at line #184
In last PR we are using different module file for rep pool to rep pool image migration, now i changed some code and make single module which will take care of rep pool to rep pool as well as ec pool to ec pool migration.